openapi-mock/server/mock.js

```diff
@@ -17,20 +17,19 @@ module.exports = async function (config) {
     var app = express()
     for (var i = 0; i < config.apis.length; i++) {
         var api = config.apis[i];
-        if (api.type = "openapi") {
+        if (!api.type || api.type == "openapi") {
             createOauthEndpoint(api, app);
             createConsole(api, app);
 
             var spec = loadSpec(api)
-
             if (spec.openapi) {
                 var jsonSpec = await transformSpec(api)
+
                 var specString = jsonSpec.stringify({ syntax: "yaml" })
                 writeSpec(specString, api, i)
                 spec = loadSpec(api)
-                spec.basePath = api.baseurl
             }
-
+            spec.basePath = api.baseurl

```

Comment:
Please make the baseurl optional and check here if it is defined at all. By default all should be taken from spec file and with config you can override.